backend/cellguide/pipeline/computational_marker_genes/types.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from typing import Dict
+
+
+@dataclass
+class ComputationalMarkerGenes:
+    me: float
+    pc: float
+    marker_score: float
+    symbol: str
+    name: str
+    groupby_dims: Dict[str, str]
```

Comment:
nit: in the future you can just use the primitive `dict[str, str]` here. previously, the benefit of using `Dict` from the `typing` package was that it would allow you to type the key/value pairs. as of some version of python recently (i forget exactly, but within the last few years?) the primitive `dict` (and `list`) support typing of the key/value pairs. the benefit of primitive types is that it's one fewer thing for you to import, but there's also no real harm in importing from the `typing` package